### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by lowks**
Simplify `.travis.yml` using native elixir support.

### CODE REVIEW

**Comment by josevalim** [comment]
It broke travis. :)

**Comment by lowks** [comment]
Sorry. Re did it.

**Comment by ericmj** [comment]
Travis is broken unrelated to this PR.

File: .travis.yml [hunk 1]
```diff
@@ -14,20 +12,16 @@ install:
...
-  - mix local.hex --force
-  - mix deps.get
   - echo "local all postgres trust" | sudo tee /etc/postgresql/$PGVERSION/main/pg_hba.conf
   - echo "host all postgres 127.0.0.1/32 trust" | sudo tee -a /etc/postgresql/$PGVERSION/main/pg_hba.conf
   - echo "host all postgrex_md5_pw 127.0.0.1/32 md5" | sudo tee -a /etc/postgresql/$PGVERSION/main/pg_hba.conf
   - echo "host all postgrex_cleartext_pw 127.0.0.1/32 password" | sudo tee -a /etc/postgresql/$PGVERSION/main/pg_hba.conf
   - sudo service postgresql restart
 script:
-  - mix test
+  - Yes y | mix test
```
Comment (ericmj) [inline .travis.yml]: mix test isn't needed

**Comment by ericmj** [comment]
Please rebase to a single commit and I will merge after travis passes.

**Comment by ericmj** [comment]
Seems like there are still CI failures after master passes.

**Comment by ericmj** [comment]
If you move install to before_install it should work. The install step replaces the default install step that installs hex.

**Comment by lowks** [comment]
Okay let me try that. Did not know install can come before before_install

**Comment by lowks** [comment]
Thanks Eric, that works! I have merged all the commits into one as well.

**Pull request merged**